client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -194,17 +195,18 @@ const PersonalInfoTab: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
                             control={control}
                             name={PersonalInfoDataContextFields.PHONE_NUMBER}
                             render={(props) => (
-                                <TextField
-                                    id={PHONE_LABEL}
+                                <NumericTextField
+                                    name={props.name}
                                     test-id='personalDetailsPhone'
                                     className={classes.phoneInput}
                                     value={props.value}
-                                    onChange={(newValue) => (
-                                        props.onChange(newValue.target.value)
+                                    onChange={(newValue: string) => (
+                                        props.onChange(newValue)
                                     )}
                                     placeholder={PHONE_LABEL}
-                                    error={errors[PersonalInfoDataContextFields.PHONE_NUMBER]}
-                                    label={errors[PersonalInfoDataContextFields.PHONE_NUMBER] ? errors[PersonalInfoDataContextFields.PHONE_NUMBER]?.message : 'טלפון*'}
```

Comment:
I think you should keep 'טלפון*' label

Reply:
I brought them all back. We don't need the error logic in here anymore so I only provide the normal label.